pkg/network/protocols/http2/telemetry.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build (windows && npm) || linux_bpf
+
+package http2
+
+import (
+	"fmt"
+
+	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type kernelTelemetry struct {
+	// metricGroup is used here mostly for building the log message below
+	metricGroup *libtelemetry.MetricGroup
+
+	// http2requests             Count of HTTP/2 requests seen
+	// http2responses            Count of HTTP/2 responses seen
+	// endOfStreamEOS            Count of END_OF_STREAM flags seen
+	// endOfStreamRST            Count of RST flags seen
+
+	// pathSizeBucket0           Count of path sizes is less or equal than 120
+	// pathSizeBucket1           Count of path sizes between 121-130 bytes
+	// pathSizeBucket2           Count of path sizes between 131-140 bytes
+	// pathSizeBucket3           Count of path sizes between 141-150 bytes
+	// pathSizeBucket4           Count of path sizes between 151-160 bytes
+	// pathSizeBucket5           Count of path sizes between 161-179 bytes
+	// pathSizeBucket6           Count of path is larger or equal to 180
+
+	// strLenGreaterThanFrameLoc Count of times we couldn't retrieve the path due to reaching the end of the frame.
+	// maxInterestingFrames		 Count of times we reached the max number of frames per iteration.
+	// maxFramesToFilter		 Count of times we have left with more frames to filter than the max number of frames to filter.
```

Comment:
please move documentation to be close to the fields
instead of 
```
// fieldA doc....
// fieldB doc...
...
// fieldZ doc...

A ...
B ...
...
Z ...
```

do
```
// fieldA doc...
A ...

// fieldB doc...
B ...